src/statement.ts

```diff
@@ -191,22 +196,22 @@ export class Statement<S = unknown> {
    * Instead it returns the object directly. If the query returns no
    * rows, then first() will return null.
    * @param colName If provided, filter results to the provided column.
-   * @param opts Additional options to control execution.
+   * @param controller An optional object used to control receipt polling behavior.
    */
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   async first<T = S, K extends keyof T = keyof T>(): Promise<T>;
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   async first<T = S, K extends keyof T = keyof T>(
     colName: undefined,
-    opts?: SignalAndInterval
+    controller?: PollingController
   ): Promise<T>;
   async first<T = S, K extends keyof T = keyof T>(
     colName: K,
-    opts?: SignalAndInterval
+    controller?: PollingController
   ): Promise<T[K] | null>;
   async first<T = S, K extends keyof T = keyof T>(
     colName?: K,
-    opts: SignalAndInterval = {}
+    controller?: PollingController
```

Comment:
Is there any reason this was not optional and was instead an empty object by default. I made this change in a bunch of places.

Reply:
Providing a default value _makes_ it effectively optional. If you look at the typescript signature, it includes the `?`. That way, you don't have to double check that a) opts is provided, and then b) if a given property on opts is provided. With chained ? this isn't as much of an issue, but it seemed to lead to cleaner code.

Reply:
Yeh that's true, with those chained `?` it is pretty clean. Ok!